go-controller/pkg/node/node.go

```diff
@@ -279,12 +280,23 @@ func (n *OvnNode) Start(wg *sync.WaitGroup) error {
 	n.WatchEndpoints()
 
 	// start the cni server
-	cniServer := cni.NewCNIServer("", kclient.KClient)
-	err = cniServer.Start(cni.HandleCNIRequest)
+	n.cniServer = cni.NewCNIServer("", kclient.KClient)
```

Comment:
is not better to plumb the informer down?
is just replacing the lister here by the informer 
https://github.com/ovn-org/ovn-kubernetes/pull/1832
and also we keep the code independent

Reply:
@aojea Wouldn't we have to do a new setup for the factory using that informer or something?

Reply:
thinking out loud, is not that the CNIserver is going to be decoupled of the node, right?

Reply:
See if the current code looks like what you expect?